docs/docs/60-new-docs/50-user-guide/20-how-to-guides/20-managing-credentials.md

```diff
@@ -23,11 +23,6 @@ This section presents an overview of how these credentials can be managed.
 
 ## Credentials as Kubernetes `Secret` Resources
 
-:::caution
-Kargo formerly borrowed its general credential-management approach from Argo CD,
-but has since diverged.
-:::
```

Comment:
Yup. I think was valuable to call this out once upon a time and I think we're past that.